gax-grpc/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>gax-grpc</artifactId>
```

Comment:
Why doesn't this have `<version>` unlike the `pom.xml` in `gax-httpjson`? Does this inherit the same version from the parent? (Didn't know that, if that's the case.)

But currently, we maintain versions separately for all sub-modules in `versions.txt`. However, if it's obvious that all of them must always match, I am down to maintaining only one version.

Reply:
Yes, the version is inherited from the parent. However, for now, we should probably just have it declared explicitly.